description: disable flags that are introduced after clang 7
author: Michael Gilbert <mgilbert@debian.org>

--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1515,24 +1515,6 @@ config("default_warnings") {
           # TODO(thakis): Only for no_chromium_code? http://crbug.com/912662
           "-Wno-ignored-pragma-optimize",
 
-          # TODO(https://crbug.com/989932): Evaluate and possibly enable.
-          "-Wno-implicit-int-float-conversion",
-
-          # TODO(https://crbug.com/999886): Clean up, enable.
-          "-Wno-final-dtor-non-final-class",
-
-          # TODO(https://crbug.com/1016947) Clean up, enable.
-          "-Wno-bitwise-conditional-parentheses",
-
-          # TODO(https://crbug.com/1016945) Clean up, enable.
-          "-Wno-builtin-assume-aligned-alignment",
-
-          # TODO(https://crbug.com/1028110): Evaluate and possible enable.
-          "-Wno-deprecated-copy",
-
-          # TODO(https://crbug.com/1031169): Clean up and enable.
-          "-Wno-misleading-indentation",
-
           # TODO(https://crbug.com/1039697): Evaluate and possibly enable.
           "-Wno-range-loop-analysis",
         ]
--- a/third_party/pffft/BUILD.gn
+++ b/third_party/pffft/BUILD.gn
@@ -22,7 +22,6 @@ config("common_config") {
 
 static_library("pffft") {
   configs += [ ":common_config" ]
-  cflags = [ "-Wno-unused-but-set-variable" ]
   sources = [
     "src/pffft.c",
     "src/pffft.h",
--- a/third_party/blink/renderer/BUILD.gn
+++ b/third_party/blink/renderer/BUILD.gn
@@ -49,7 +49,7 @@ config("inside_blink") {
       "-Wno-implicit-int-conversion",
     ]
 
-    if (!is_chromeos || default_toolchain != "//build/toolchain/cros:target") {
+    if (false) {
       cflags += [
         "-Wno-enum-enum-conversion",
         "-Wno-enum-float-conversion",
